spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -205,6 +205,18 @@
         ).to be >= 0.0
       end
 
+      context 'when an enrollment does not have a unique ID' do
+        it 'generates a backwards-compatible unique ID' do
```

Comment:
Should there be code changes to the job implementation? I only see that the unique ID would be assigned at initial creation.

Reply:
Not to the job implementation but there is a change in this PR in the enrollment_helper file. I'm going to add more details to the PR description and a new commit with the missing change

Reply:
The job already updates the unique ID if necessary [here](https://github.com/18F/identity-idp/blob/31108ac1d5f760c0a3470116143238ca8f7e1521/app/jobs/get_usps_proofing_results_job.rb#L75). But see a8f85c6bab1ceb71232abe939be3c9b403ec65f5 where I added another missing piece: updated enrollment_helper to use the value of `enrollment.unique_id` if it exists